File: src/main/java/com/cds/command/ConsoleCommandHandler.java
package com.cds.command;

import org.bukkit.command.Command;
import org.bukkit.command.ConsoleCommandSender;
import org.bukkit.plugin.java.JavaPlugin;

import com.cds.handler.BaseHandler;

public class ConsoleCommandHandler extends BaseHandler {

    ConsoleCommandHandler(JavaPlugin plugin) {
        super(plugin);
    }

    public boolean handleCommand(ConsoleCommandSender sender, Command cmd, String label, String[] args) {
        return false;
    }
}

File: src/main/kotlin/com/cds/command/ConsoleCommandHandler.kt
package com.cds.command

import com.cds.CDSPlugin
import com.cds.handler.BaseHandler
import org.bukkit.command.Command
import org.bukkit.command.ConsoleCommandSender

public class ConsoleCommandHandler(plugin: CDSPlugin) : BaseHandler(plugin) {

    fun handleCommand(
            sender: ConsoleCommandSender,
            cmd: Command,
            label: String,
            args: Array<String>
    ): Boolean {
        // Case Insensitive Command
        val ciCmd = cmd.getName().lowercase()
        return when (ciCmd) {
            else -> false
        }
    }
}
